Hexastore.Web/EventHubs/EventSender.cs

```diff
@@ -50,10 +49,15 @@ public async Task SendMessage(StoreEvent storeEvent)
             }
 
             try {
-                storeEvent.PartitionId = storeEvent.StoreId.GetHashCode() % _storeConfig.EventHubPartitionCount;
+                int ehPartitionId;
+                if (!string.IsNullOrEmpty(storeEvent.PartitionId)) {
```

Comment:
Instead of passing a partition ID, I'd opt for got a hash (partition hash, object hash, device key, not sure what to call it) that specifies everything with this hash should go to the same place, but then we can jump hash it ourselves, or redistribute across different queues/partitions if one partition turns out to be noisier than the rest

Reply:
Saw that it's used as more of a key than an ID lower down with EH, so I guess this is just more of a naming misunderstanding.

Reply:
I think so. The way you described it is the way it actually works. Open to a better name.